include.js

```diff
@@ -6275,6 +6275,13 @@ let cards = [
     author: 'Katana',
     githubLink: 'https://github.com/katanaji'
   }
```

Comment:
Please add the missing comma between curly brackets to avoid syntax errors.

Reply:
Hi Sophia, thanks for pointing this out - it should be resolved now